src/cpp/session/modules/SessionRCompletions.R

```diff
@@ -2554,6 +2554,10 @@ assign(x = ".rs.acCompletionTypes",
    
 })
 
+.rs.addJsonRpcHandler("console_follow_hyperlink", function(url, link_params){
```

Comment:
Should probably live elsewhere

Reply:
```suggestion
.rs.addJsonRpcHandler("console_follow_hyperlink", function(url, linkParams)
{
```

We generally use braces on their own newline, + camel case for variable names.

Reply:
We have an empty 'SessionConsole.R' module here:

https://github.com/rstudio/rstudio/blob/main/src/cpp/session/modules/SessionConsole.R

Maybe a good fit there? Or, a new `SessionVirtualConsole.R` module?